linode/instanceip/tmpl/template.go

```diff
@@ -10,6 +10,7 @@ type TemplateData struct {
 	Label            string
 	ApplyImmediately bool
 	Region           string
+	Address          string
```

Comment:
It doesn't look like this field is referenced in any of the templates; would it make sense to drop it from this PR?

Reply:
Yes I just confirmed it can be dropped. I will remove it from the template file.